auto_round/utils.py

```diff
@@ -294,7 +294,6 @@ def get_module(module, key):
     for name in name_list:
         if hasattr(module, name):
             module = getattr(module, name)
-            module = module
```

Comment:
It's unreasonable to return the orig model if the key could not found.  Please change to return None instead